### PR TITLE
Update docker/login-action

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
Use [the latest version](https://github.com/docker/login-action/releases).

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/publish_image_chart.yaml [hunk 1]
```diff
@@ -55,14 +55,14 @@ jobs:
 
       # Login to push container images.
       - name: Log in to the container registry
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```
Comment (khanhtc1202) [inline .github/workflows/publish_image_chart.yaml:58]: could you add comment to show which version of login-action do we use for now 🙏

Comment (knanao) [reply .github/workflows/publish_image_chart.yaml:58]: Sure

**Review by nghialv — APPROVED** [review]
👍

**Review by khanhtc1202 — APPROVED** [review]
👍

**Pull request merged**